docs/src/main/asciidoc/native-reference.adoc

```diff
@@ -153,6 +153,381 @@ $ quarkus-project-0.1-SNAPSHOT-runner -XX:PrintFlags=
   -XX:±VerboseGC                               Print more information about the heap before and after each collection. Default: - (disabled).
 ----
 
+[[rss]]
+=== Resident Set Size (RSS)
+
+As described in the
+xref:performance-measure.adoc[Measuring Performance guide],
+the footprint of Quarkus applications is measured using the resident set size (RSS).
+This is also applicable to native applications,
+but the runtime engine that manages the footprint is GraalVM's SubstrateVM rather than the JVM.
```

Comment:
I guess just say `GraalVM`?

Reply:
What do you think about something like this?

```suggestion
but the runtime engine that manages the footprint in this case is built in the native executable itself rather than the JVM.
```